requirements-integration.in

```diff
@@ -1,6 +1,7 @@
 aiohttp
 jinja2
-juju<3.1
+# Pinning to <3.0 due to compatibility with the 2.9 controller version and 3.0 not being maintained anymore
+juju<3.0
```

Comment:
Although this is the version we should've used back then (`juju<3.0`), I'm more in favour of not changing the state of this if it is not really needed. Was it giving you issues during integration testing?